src/main/java/com/ryszardpanda/GitHubReposFetcher/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,19 @@
+package com.ryszardpanda.GitHubReposFetcher.exception;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+    @ExceptionHandler({LocalRepositoryNotFoundException.class})
+    public ResponseEntity<ErrorMessage> handleLocalRepositoryNotFoundException(
```

Comment:
Jak uzywamy RestControllerAdvice to tak jakbysmy uzywali RestControllera nie ma wtedy potrzeby zwracac ResponseEntity poniewaz wystarczy zwrocic to co ma byc w body a statusem mozemy sterować przy uzyciu adnotacji @ResponseStatus nad metoda